intuit.css

```diff
@@ -0,0 +1,163 @@
+.body_style {
+  background-color: #dfdfdf;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.sign-in-part {
+  width: 23%;
```

Comment:
Please remove that `width: 23%` to fix the error @uche-inyama mentioned above.

Reply:
removed the width: 23% property